Makefile

```diff
@@ -0,0 +1,31 @@
+
+.ONESHELL:
+	SHELL:=/bin/bash 
+
+.PHONY: create-env install documentation test
+
+default:
+	@echo "Call a specific subcommand: create-env,install,documentation,test"
+
+create-env:
```

Comment:
I would suggest to change the target name to the venv directory name (`.venv`) here to enable make dependency checking features, or to create another non-phony target
```
.venv:
   ...
```

Reply:
it's change